fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraFixity.java

```diff
@@ -114,4 +139,23 @@ protected Session session() {
     protected String externalPath() {
         return externalPath;
     }
+
+    private String setUpBaseURL(final UriInfo uriInfo) {
+        try {
+            final URI baseURL = uriInfo.getBaseUri();
+            LOGGER.debug("setting baseURL = " + baseURL.toString());
+            return baseURL.toString();
+        } catch (Exception ex) {
+            LOGGER.warn("Error setting baseURL", ex);
+        }
+        return "";
+    }
+
+    private String setUpAgent(final HttpHeaders headers) {
+        if (!StringUtils.isBlank(headers.getHeaderString("user-agent"))) {
+            return headers.getHeaderString("user-agent");
```

Comment:
I think this is the same issue we were discussing yesterday -- we do want to capture the userAgent that triggers the fixity check, and we need to get that from the REST API endpoint since that's the only place it's accessible.  We also want the userID of the authorized user, but we can get that at a lower level (see https://github.com/fcrepo4/fcrepo4/pull/765/files#diff-49c1e2bcddbf58f2a8792f91bc8f3545R125).